server/follow/index.js

```diff
@@ -0,0 +1,118 @@
+'use strict'
+
+const serie = require('fastseries')()
+
+const {
+  follow: followSchema,
+  unfollow: unfollowSchema,
+  followers: followersSchema
+} = require('./schemas')
+const FollowService = require('./FollowService')
+
+module.exports = function (fastify, opts, next) {
+  // See user/index.js for some little explainations
+  serie(
+    fastify,
+    [
+      registerEnv,
+      registerRedis,
+      decorateWithTweetService,
+      decorateWithUserClient,
+      registerRoutes
+    ],
+    opts,
+    next
+  )
+}
+
+function registerEnv (data, done) {
+  const envOpts = {
+    schema: {
+      type: 'object',
+      required: [ 'FOLLOW_REDIS_URL', 'USER_MICROSERVICE_BASE_URL' ],
+      properties: {
+        FOLLOW_REDIS_URL: { type: 'string', default: '127.0.0.1' },
+        USER_MICROSERVICE_BASE_URL: { type: 'string', default: 'http://localhost:3001' }
+      }
+    },
+    data: data
+  }
+  this.register(require('fastify-env'), envOpts, done)
+}
+
+function registerRedis (a, done) {
+  this.register(require('fastify-redis'), {
+    host: this.config.FOLLOW_REDIS_URL
+  }, done)
+}
+
+function decorateWithTweetService (a, done) {
+  const followService = new FollowService(this.redis)
+  this.decorate('followService', followService)
+  done()
+}
+
+const request = require('request-promise-native')
```

Comment:
I would prefer if you use got here.

Reply:
Yes. after the sermver break, I want to leave request out.